src/test/java/SalesforceModules/OppurtunitiesUtil.java

```diff
@@ -415,18 +420,88 @@ public static void clickOnSalesforceCPQ() {
     public static void verifyOpportunityStage() throws InterruptedException {
         KeywordUtil.delay(5000);
         String stageStatus = KeywordUtil.getVisibleText(OppurtunitiesObject.opportunityStage);
-        if (KeywordUtil.isWebElementVisible(OppurtunitiesObject.opportunityStage,"Closed Won Stage Displayed")){
-            RunCukesTest.logger.log(LogStatus.PASS,HTMLReportUtil.passStringGreenColor("Verify Status: "+stageStatus));
-        }
-        else{
-            RunCukesTest.logger.log(LogStatus.FAIL,"Status is not showing: "+stageStatus);
+        if (KeywordUtil.isWebElementVisible(OppurtunitiesObject.opportunityStage, "Closed Won Stage Displayed")) {
+            RunCukesTest.logger.log(LogStatus.PASS, HTMLReportUtil.passStringGreenColor("Verify Status: " + stageStatus));
+        } else {
+            RunCukesTest.logger.log(LogStatus.FAIL, "Status is not showing: " + stageStatus);
 
         }
     }
 
 
+    //(OppurtunitiesObject.opportunityStage,"Stage status Closed Won"));
```

Comment:
do we need this line? if not then please remove it

Reply:
that line is removed.